src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -150,6 +183,7 @@ function withPostGraphileContextFromReqResGenerator(
         jwtToken,
         pgSettings,
         explain: allowExplain && req.headers['x-postgraphile-explain'] === 'on',
+        explainOptions: parseExplainOptions(req),
```

Comment:
Let's merge explain/explainOptions together. We don't need to support any legacy here since PostGraphile and PostGraphiQL are deeply integrated and tied to the exact same versions, so unified changes are fine - no need for backwards compatibility.

```suggestion
        explain: allowExplain && parseExplainOptions(req),
```